test/functional/actions/setCustomerIds.spec.js

```diff
@@ -31,27 +30,18 @@ const mockExtensionSettings = {
   ]
 };
 
-const instanceNameField = spectrum.select(Selector("[name=instanceName]"));
-const addCustomerIdButton = spectrum.button(Selector("#addCustomerId"));
```

Comment:
Probably don't need the id property on the add customer id button anymore.

Reply:
Where did you see one? I believe it was removed: https://github.com/adobe/reactor-extension-alloy/pull/65/files#diff-73b166646fdf3d8b15c143f08ada5d96L40

Reply:
Sorry I was mistaken.